server/sonar-web/src/main/js/apps/issues/controller.js

```diff
@@ -146,7 +146,7 @@ export default Controller.extend({
       _.extend(filter, { assignees: '__me__' });
     }
     const route = [];
-    _.map(filter, function (value, property) {
+    _.map(filter, (value, property) => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)